internal/services/managedapplications/managed_application_resource_test.go

```diff
@@ -153,6 +153,58 @@ func TestAccManagedApplication_tags(t *testing.T) {
 	})
 }
 
+func TestAccManagedApplication_identity(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_managed_application", "test")
+	r := ManagedApplicationResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
+		{
+			Config: r.systemAssignedIdentity(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
```

Comment:
Can you add more steps to test it can change from system assigned identity to user assigned identity, then change back to none identity?